Unit3/Module3.1/Mini-project3.1a/README.md

```diff
@@ -0,0 +1,10 @@
+# Mini Project 3.1a: Add a Validation Message
+
+##### Credit to OpenClassrooms
+![Become](https://github.com/OCclassprojects/logo/blob/master/fav-icon.png?raw=true)
+
+### Mini Project Instructions
+
+1. Open the [project page on OpenClassrooms](https://openclassrooms.com/en/courses/4286486-build-web-apps-with-reactjs/4317901-practice-add-a-validation-message)
+
+2. Create a new branch and then new Pull Request and assign it to your coach for review.
```

Comment:
Here it is probably helpful to tell them what specific files they should be putting into the PR (referring to files that they would be building out according to OpenClassroom instructions).